Dockerfile

```diff
@@ -0,0 +1,82 @@
+##################################
+# Build
+##################################
+FROM node:14 as build
+
+RUN mkdir -p /opt/build;
+
+WORKDIR /opt/build
+
+# Would be a bit simpler if the code was inside a top-level src folder
```

Comment:
Mind opening an issue to track refactoring all the source files into one directory?

Reply:
Sure thing. Will do.

Reply:
Done: https://github.com/nteract/commuter/issues/293